packages/editor/src/panels/assets/resources.tsx

```diff
@@ -162,6 +162,12 @@ function ResourceFile({ resource }: { resource: StaticResourceType }) {
     if (preview) preview(getEmptyImage(), { captureDraggingState: true })
   }, [preview])
 
+  const clickState = useMutableState(ClickPlacementState)
+  let isSelected = clickState.selectedAsset.value === resource.url
+  useEffect(() => {
```

Comment:
the use effect here is redundant, isSelected can then be a const

Reply:
Ah, right! Made the requested change. 👍